src/logic/currencyValues/store/selectors/index.js

```diff
@@ -1,13 +1,40 @@
 // @flow
 
 import { List } from 'immutable'
+import { type OutputSelector, createSelector } from 'reselect'
 
+import type { CurrencyValuesEntry, CurrencyValuesProps } from '~/logic/currencyValues/store/model/currencyValues'
 import { CURRENCY_VALUES_KEY } from '~/logic/currencyValues/store/reducer/currencyValues'
+import { safeParamAddressFromStateSelector } from '~/routes/safe/store/selectors'
 import { type GlobalState } from '~/store'
 
-export const currencyValuesListSelector = (state: GlobalState) =>
-  state[CURRENCY_VALUES_KEY].get('currencyBalances') ? state[CURRENCY_VALUES_KEY].get('currencyBalances') : List([])
+export const currencyValuesSelector = (state: GlobalState): CurrencyValuesEntry =>
+  state[CURRENCY_VALUES_KEY].get('currencyValues')
```

Comment:
By removing the duplicated `currencyValues` key, then this should change to:

```suggestion
  state[CURRENCY_VALUES_KEY]
```